src/api/user/userController.ts

```diff
@@ -0,0 +1,19 @@
+import type { Request, RequestHandler, Response } from "express";
+
+import { userServiceInstance } from "@/api/user/userService";
```

Comment:
Could you update to 'userService'? similar to the userController class. 

Im liking the pascal case for the classes. 😀